crates/core/asset/src/asset/registry.rs

```diff
@@ -146,23 +146,29 @@ impl Builder {
             for _d in displays.iter() {
                 display_to_base.push(base_index);
             }
-            display_regexes.push(displays.iter().map(|d| Regex::new(d).unwrap()).collect());
+            display_regexes.push(
+                displays
+                    .iter()
+                    .map(|d| Regex::new(d).expect("unable to parse display regex"))
```

Comment:
The `expect` argument should be a description of the invariant that's expected to be upheld, right? So this should be
```
.expect("display regex must be valid")
```
but why is that the case?

Reply:
> The `expect` argument should be a description of the invariant that's expected to be upheld, right?

I am not sure. Our codebase is inconsistent on this and I've also heard it's better to write `expect` messages like you would an `anyhow::anyhow!("...")` error message due to the way they're printed in logs. It would be nice to pick one and stick with it.